src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/ResourceHeatMapGraphTest.java

```diff
@@ -151,6 +150,12 @@ public static void shutdown() {
     clientServers.getHttpServer().stop(0);
     clientServers.getNetServer().stop();
     clientServers.getNetClient().stop();
+
+    try {
+      Thread.sleep(1000);
```

Comment:
It isn't strictly necessary, I'm mimicking similar logic from RcaControllerTest which does this sleep after stopping its clients and servers.

Reply:
nit: Maybe it’s a good idea to abstract the shutdown logic with sleep into its own method so that we don’t have to carry the sleep change wherever we use shutdown?